create_infotexts.py

```diff
@@ -29,13 +30,15 @@ def load_places_mapping():
     places_df = places_df.set_index("Nyckelord")
     places_df.columns = ["freq", "commons", "wikidata"]
 
+    places_df.replace("-", np.nan)
+
     places_dict = {}
 
     for index, row in places_df.iterrows():
         places_dict[index] = {}
         places_dict[index]["commons"] = row["commons"]
         places_dict[index]["wikidata"] = row["wikidata"]
-
+    print(places_dict)
```

Comment:
I don't understand why you see that printout - it's not in my current code in the branch. Inshallah it won't be there when I commit again...